pkg/watch/health.go

```diff
@@ -212,7 +214,25 @@ func (p *PodWatch) checkHealth() {
 // Given the result of a status check this method
 // creates a health.Result for that node/service/result
 func (sc *StatusChecker) Check() (health.Result, error) {
-	return sc.resultFromCheck(sc.StatusCheck())
+	if sc.URI != "" {
+		return sc.resultFromCheck(sc.StatusCheck())
+	} else {
+		// "unknown" is probably more accurate, but automated tools can't handle an app that is
+		// always non-"passing". For instance, p2-replicate by default waits for a node to
+		// become "passing" before it considers the deployment a success.
+		//
+		// TODO: P2 has the capacity to check whether the app's process is running. This would
+		// make a great default status check! However, that information isn't easily accessible
+		// over here in the watch package. It would take a lot of refactoring to make this
+		// happen.
```

Comment:
Except that frequently people deploy apps intended to not run at all, even if they have running processes - basically using P2 as a tar delivery mechanism. Perhaps the right solution is to provide a generic runit monitor sidekick that can be added to a pod manifest